llvm/test/CodeGen/AArch64/optimize_combine_large_shifts.ll

```diff
@@ -0,0 +1,37 @@
+; RUN: update_llc_test_checks < %s -verify-machineinstrs -mtriple=aarch64-none-linux-gnu -mattr=neon | FileCheck %s
```

Comment:
I just realized how stupid this change is. I will work on figuring out update_llc_test_checks today, I had a long day yesterday and wanted to get some work done but my brain was out of it.

Reply:
I rolled back the line until I figure it out